<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLanguageSelectorGUI</class>
 <widget class="QWidget" name="QtLanguageSelectorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Language Installer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="InstallTab">
      <attribute name="title">
       <string>Install</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="selectLanguageLabel">
         <property name="text">
          <string>Select language to install:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listViewLanguagesInst"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxTr">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Translations</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxIm">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Input methods</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxSpell">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Spellchecking and writing aids</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBoxFonts">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Extra fonts</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="ComponentsLabel">
           <property name="text">
            <string>Components:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UninstallTab">
      <attribute name="title">
       <string>Uninstall</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="selectLanguageLabel_2">
         <property name="text">
          <string>Select language to uninstall:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listViewLanguagesUninst"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SystemDefaultTab">
      <attribute name="title">
       <string>Set System Language</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="defaultSystemLabel">
         <property name="text">
          <string>Default system language:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listBoxDefaultLanguage"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelInputMethod">
           <property name="text">
            <string>Keyboard input method:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxInputMethod"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
